src/Enum/ValueTypeEnum.php

```diff
@@ -32,5 +32,5 @@ final class ValueTypeEnum
     /**
      * @var string
      */
-    public const TYPE_INT = 'int';
+    public const TYPE_INTEGER = 'integer';
```

Comment:
Please rather use the short form for bool above! We never use the long versions on code elements as the short ones are clear enough already. They are only for textual usage.

Reply:
changed